packages/backend-lalr/src/Happy/Backend/LALR/ProduceCode.lhs

```diff
@@ -724,18 +835,45 @@ action array indexed by (terminal * last_state) + state
 >           . interleave' "," (map shows table)
 >           . str "\n\t])\n\n"
 
+>    produceExpListArray :: CodeGen e => NameContext e [[DecT e]]
```

Comment:
I don’t really understand why this function is returning something specific instead of just generating code as all other functions do?